sovtokenfees/sovtokenfees/main.py

```diff
@@ -13,81 +26,53 @@
-
+    token_ledger = node.db_manager.get_ledger(TOKEN_LEDGER_ID)
+    token_state = node.db_manager.get_state(TOKEN_LEDGER_ID)
     node.write_req_validator.auth_map.update(sovtokenfees_auth_map)
 
-    token_authnr = node.clientAuthNr.get_authnr_by_type(TokenAuthNr)
-    if not token_authnr:
-        raise ImportError('sovtoken plugin should be loaded, '  # noqa
-                          'authenticator not found')
-    token_req_handler = node.get_req_handler(ledger_id=TOKEN_LEDGER_ID)
-    if not token_req_handler:
-        raise ImportError('sovtoken plugin should be loaded, request '  # noqa
-                          'handler not found')
+    fees_tracker = BatchFeesTracker()
+    token_tracker = LedgerUncommittedTracker(token_state.committedHeadHash,
+                                             token_ledger.uncommitted_root_hash,
+                                             token_ledger.size)
 
-    # `handle_xfer_public_txn` in `TokenReqHandler` checks if the sum of inputs match
-    # exactly the sum of outputs. Since the check to match inputs and outputs is done
-    # during fees handling the check is avoided in `TokenReqHandler` by monkeypatching
-    # `handle_xfer_public_txn` to do nothing.
-    token_req_handler.handle_xfer_public_txn = lambda _: None
+    def postCatchupCompleteClbk():
+        token_tracker.set_last_committed(token_state.committedHeadHash,
+                                         token_ledger.uncommitted_root_hash,
+                                         token_ledger.size)
 
-    token_ledger = token_req_handler.ledger
-    token_state = token_req_handler.state
-    utxo_cache = token_req_handler.utxo_cache
-    fees_authnr = FeesAuthNr(node.getState(DOMAIN_LEDGER_ID), token_authnr)
-    fees_req_handler = StaticFeesReqHandler(node.configLedger,
-                                            node.getState(CONFIG_LEDGER_ID),
-                                            token_ledger,
-                                            token_state,
-                                            utxo_cache,
-                                            node.getState(DOMAIN_LEDGER_ID),
-                                            node.bls_bft.bls_store, node,
-                                            node.write_req_validator,
-                                            ts_store=node.getStateTsDbStorage())
-    origin_token_clb = node.ledgerManager.ledgerRegistry[TOKEN_LEDGER_ID].postCatchupCompleteClbk
-    node.ledgerManager.ledgerRegistry[TOKEN_LEDGER_ID].postCatchupCompleteClbk = \
-        functools.partial(postCatchupCompleteClb, origin_token_clb)
+    utxo_cache = node.db_manager.get_store(UTXO_CACHE_LABEL)
 
-    origin_token_post_added_clb = node.ledgerManager.ledgerRegistry[TOKEN_LEDGER_ID].postTxnAddedToLedgerClbk
-
-    def filter_fees(ledger_id: int, txn: Any):
-        origin_token_post_added_clb(ledger_id, txn, get_type(txn) != FeesTransactions.FEES.value)
+    register_req_handlers(node, fees_tracker)
```

Comment:
Please, move this bootstrap things into separate function/method

Reply:
ok